bin/check_license.sh

```diff
@@ -2,13 +2,28 @@
 SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
 ROOTDIR=$SCRIPTPATH/..
 cd $ROOTDIR
+# ';' separated list of files, on which we don't want to run this script.
+WHITELIST_FILES="pkg/ctrlz/assets.gen.go;"
```

Comment:
There are other places where assets will run into the same problem as well.
Perhaps it is better to have a general rule that deals with this, similar to some of the grep checks below.

Reply:
Thanks.. Fixed